crates/driver/src/tests/cases/asset_flow.rs

```diff
@@ -23,10 +23,21 @@ async fn matrix() {
                     surplus_fee: order::SellAmount(DEFAULT_SURPLUS_FEE.into()),
                 },
             ] {
+                let solver_fee = match kind {
+                    order::Kind::Market => None,
+                    order::Kind::Limit { .. } => Some(0.into()),
```

Comment:
Can we test asset-flow with a solver fee?

Reply:
Done.